### ISSUE TITLE
Will the code for lightweight fine-tuning be open source?

### ISSUE THREAD

**Issue by Aurora-1412** [Alibaba-NLP/ZeroSearch#25]
Thanks for your innovative work! Could you please to let me know more details in lightweight fine-tuning, such as the SFT framework used, hyper-parameters, and how to use positive and negative samples for fine-tuning?

**Comment by sunhaonlp** [comment]
Hi, thanks for your interest! We performed lightweight fine-tuning using [[LLaMA-Factory](https://github.com/hiyouga/LLaMA-Factory)](https://github.com/hiyouga/LLaMA-Factory) with a learning rate of 1 × 10⁻⁶, a batch size of 2, and 1,000 total training steps. We also balanced the ratio of positive to negative samples, and the resulting dataset is available at [https://huggingface.co/datasets/sunhaonlp/SimulationTuning\_dataset](https://huggingface.co/datasets/sunhaonlp/SimulationTuning_dataset).
